package.json

```diff
@@ -1,179 +1,188 @@
+    "devDependencies": {
+        "@types/basic-auth": "^1.1.3",
+        "@types/bcrypt": "^3.0.0",
+        "@types/cron": "^1.7.2",
+        "@types/ejs": "^3.0.5",
+        "@types/express": "^4.17.4",
+        "@types/graphql-fields": "^1.3.3",
+        "@types/jest": "^29.4.4",
+        "@types/jsonwebtoken": "^9.0.2",
+        "@types/keyv": "^3.1.2",
+        "@types/lodash": "^4.14.155",
+        "@types/mime-types": "^2.1.0",
+        "@types/multer": "^1.4.5",
+        "@types/node": "^12.12.31",
+        "@types/node-mailjet": "^3.3.3",
+        "@types/object-hash": "^3.0.2",
+        "@types/papaparse": "^5.0.3",
+        "@types/qrcode": "^1.5.1",
+        "@types/react-select": "^3.0.12",
+        "@types/uuid": "^8.3.4",
+        "@types/ws": "^8.5.3",
+        "@typescript-eslint/eslint-plugin": "^6.4.1",
+        "@typescript-eslint/parser": "^6.4.1",
+        "apidoc": "^0.22.1",
+        "concurrently": "^8.2.0",
+        "eslint": "^8.2.0",
+        "eslint-import-resolver-typescript": "^3.5.2",
+        "eslint-plugin-import": "^2.26.0",
+        "eslint-plugin-lernfair-lint": "./linter/",
+        "jest": "^29.5.0",
+        "lint-staged": "^14.0.1",
+        "nodemon": "^3.0.1",
+        "pre-commit": "^1.2.2",
+        "prettier": "^2.7.1",
+        "prisma": "4.15.0",
+        "ts-jest": "^29.0.5",
+        "typegraphql-prisma": "0.26.0",
+        "typescript": "^4.3.5"
     },
-    "collectCoverageFrom": [
-      "**/*.(t|j)s"
+    "optionalDependencies": {
+        "corona-school-matching": "1.3.0",
+        "html-pppdf": "^1.0.1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/corona-school/backend"
+    },
+    "keywords": [
+        "node",
+        "heroku",
+        "express"
     ],
-    "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
-  },
-  "nodemonConfig": {
-    "verbose": true,
-    "ignore": [
-      ".git",
-      "node_modules/**/node_modules"
+    "license": "MIT",
+    "pre-commit": [
+        "lint-staged"
     ],
-    "ext": "js,json"
-  }
+    "jest": {
+        "moduleFileExtensions": [
+            "js",
+            "json",
+            "ts"
+        ],
+        "preset": "ts-jest/presets/default-esm",
+        "extensionsToTreatAsEsm": [
+            ".ts"
+        ],
+        "testRegex": ".*\\.spec\\.ts$",
+        "moduleNameMapper": {
+            "^(\\.{1,2}/.*)\\.js$": "$1"
+        },
+        "transform": {
+            "^.+\\.[tj]sx?$": [
+                "ts-jest",
+                {
+                    "useESM": true
+                }
+            ]
+        },
+        "collectCoverageFrom": [
+            "**/*.(t|j)s"
+        ],
+        "coverageDirectory": "../coverage",
+        "testEnvironment": "node"
+    },
+    "nodemonConfig": {
+        "verbose": true,
+        "ignore": [
+            ".git",
+            "node_modules/**/node_modules"
+        ],
+        "ext": "js,json"
+    },
+    "lint-staged": {
```

Comment:
For now we'll only run prettier inside the precommit. eslint --fix takes way too long, and a lot of errors reported cannot be fixed automatically (and then it is annoying to get a failing `git commit`)